src/test/java/UhrzeitenTest.java

```diff
@@ -18,23 +17,10 @@ public void assembleAllValidTimes() {
 		assertThat(times.size(), not(2000));
 	}
 
-	@Test(expected = IndexOutOfBoundsException.class)
+	@Test(expectedExceptions = IndexOutOfBoundsException.class)
```

Comment:
tried this change in my Eclipse project but it doesn't know "expectedExceptions"... so are you testing me here or is it a class that I should integrate in my project somehow?